models/Event.js

```diff
@@ -28,6 +32,10 @@ const EventSchema = new mongoose.Schema({
     type: String,
     required: true,
   },
+  discordName: {
+    type: String,
+    required: true,
+  },
```

Comment:
I think the discordName field is redundant. The user field (Mongo user reference) will give us all the information in regards to the user which includes the displayName (discordName).

Reply:
> I think the discordName field is redundant. The user field (Mongo user reference) will give us all the information in regards to the user which includes the displayName (discordName).

We don't have auth fully setup yet? I agree with you, but we currently do not pull the @ name currently. We'll need some way to display the @ name for the MVP.

Reply:
> So anyone can add an event without needing to authorize the first iteration MVP? They'll just add their Discord name on the form?

Ok, I was a bit wrong about current capabilities and how difficult it will be to add this functionality. We'll take a look at it shortly, but I'm approving this PR so we can move on with the refractor and then re-assess.